examples/kubernetes/sc.yaml

```diff
@@ -4,4 +4,8 @@ metadata:
   name: glusterfs-csi
   annotations:
     storageclass.beta.kubernetes.io/is-default-class: "true"
+parameters:
+  gd2url: "http://192.168.121.17:24007"
+  gd2user: "glustercli"
+  gd2userpassword: "79e8744fb454cce883dc395ecb9455296895591bd19dfdbcc50dbd2a90012ad4"
```

Comment:
Exposing credentials in the StorageClass is not a good idea. The CSI standard has [facilities to handle secret data such as credentials](https://github.com/container-storage-interface/spec/blob/master/spec.md#secrets-requirements). How that translates for the kubernetes implementation is here: https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/container-storage-interface.md#createvolumedeletevolume-credentials

For this purpose, I propose using a secret w/ 2 fields: user & key

Reply:
AFAIK, we can only pass secrets only for below operations
* CreateVolume/DeleteVolume
* ControllerPublishVolume/ControllerUnpublishVolume
* NodeStageVolume/NodeUnstageVolume
*  NodePublishVolume/NodeUnpublishVolume

we have other operations also which requires the secrets
* List Volume
* List snapshots etc

then we have to depend on the driver to have the info on how to communicate with the GD2.